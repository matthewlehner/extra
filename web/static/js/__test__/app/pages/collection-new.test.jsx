// @flow
import React from "react";
import renderer from "react-test-renderer";

import CollectionNewPage from "app/pages/collection-new";

describe("CollectionNewPage component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<CollectionNewPage />);

    expect(tree).toMatchSnapshot();
  });
});
