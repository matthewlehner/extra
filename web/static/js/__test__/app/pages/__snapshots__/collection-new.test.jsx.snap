// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionNewPage component renders correctly 1`] = `
<div>
  <header
    className="heading"
  >
    <h1>
      Create a new collection
    </h1>
  </header>
</div>
`;
